### DISCUSSION TITLE
why comment  the tileWidth and tileHeight in VcImageryProviderSingletile

### DISCUSSION THREAD

**Discussion by klren0312** [zouyaoji/vue-cesium#413] [Q&A] [answered]
### code path:
  https://github.com/zouyaoji/vue-cesium/blob/dev/packages/components/providers/single-tile/index.ts#L21

**Comment by klren0312** [comment]
https://zouyaoji.top/vue-cesium/#/zh-CN/component/providers/vc-imagery-provider-singletile
official example not work
![1689926188636](https://github.com/zouyaoji/vue-cesium/assets/10903843/3949ad78-2690-4962-a471-e6981db550cf)

**Comment by zouyaoji** [comment, marked as answer]
okay, will uncomment
